src/PackageManager/PackageManagerView.xaml.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace PackageManager
+{
+    /// <summary>
+    /// Interaction logic for PackageManagerView.xaml
+    /// </summary>
+    public partial class PackageManagerView : Window
+    {
+        public PackageManagerView()
+        {
+            InitializeComponent();
+            this.Height = (System.Windows.SystemParameters.PrimaryScreenHeight * 0.95);
```

Comment:
This size is too big and occupies almost the entire screen. Ideally, it should not be bigger than 70% of Dynamo window.